android/source/VideoLocker/src/org/edx/mobile/view/CourseInfoActivity.java

```diff
@@ -0,0 +1,52 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.BaseFragmentActivity;
+import org.edx.mobile.util.AppConstants;
+import org.edx.mobile.util.NetworkUtil;
+
+public class CourseInfoActivity extends BaseFragmentActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_find_courses);
+
+        try{
+            segIO.screenViewsTracking(getString(R.string.find_courses_title));
+        }catch(Exception e){
+            logger.error(e);
+        }
+
+        if (!(NetworkUtil.isConnected(this))) {
+            AppConstants.offline_flag = true;
```

Comment:
This can just be one line instead of five

```
AppConstants.offline_flag = !NetworkUtil.isConnected(this);
```

It's not clear to me that this should be managed by each activity at all, but that's a bigger change so let's not worry about it for now